index.js

```diff
@@ -157,10 +157,55 @@ function displayFooter() {
 }
 
 function activateSortBtns() {
-  sortBtns.forEach((btn) => btn.addEventListener("click", handleSortBtnClick));
+  sortBtns.forEach((btn) => {
+    btn.addEventListener("click", handleSortBtnClick);
+    btn.sortingState = 0;
+  });
 
   function handleSortBtnClick(e) {
-    console.log(e.target.value);
+    let sortedPosts;
+    sortBtns.forEach((btn) => {
+      if (btn.value !== e.target.value) {
+        btn.sortingState = 0;
+      }
+    });
+    postsContainer.innerHTML = "";
+    if (e.target.sortingState === 0 || e.target.sortingState === 2) {
```

Comment:
I am actually thinking, that states should be represetned as strings, like 'ascending'  and 'descending' instead of 0 and 1. Any feedback on that? 0 and 1 are not expressive enough IMO.

Reply:
Can you think of a way to use `0` for the state and use `ascending` as its name?

Reply:
no actually. ascending makes no sense. This below?
`if (e.target.sortingState === 'notSorted' || e.target.sortingState === 'onEventhInARowClick') {
      e.target.sortingState = 'onOddthInARowClick';`

that reflects the logic - if I clicked one button for the 1st, 3rd, 5th etc time, it should one state.
and if I clicked same button for the 2nd 6th 8th etc time - it should be another state.
if button isn't bothered for a while - it is notSorted? probably need smthing better than that

Reply:
And I was tlaking about not using 0, 1, 2 at all, just about replacing digits with something more readable

Reply:
Can't think of using just 2 states for now, need to tinker with it more